lib/quickbooks/model/preferences.rb

```diff
@@ -28,9 +28,28 @@ def self.create_preference_class(*attrs, &block)
       }
 
       xml_reader :sales_forms, :from => "SalesFormsPrefs", :as => create_preference_class(*%w(
-        CustomTxnNumbers? AllowDeposit? AllowDiscount? DefaultDiscountAccount? AllowEstimates? EstimateMessage?
-        ETransactionEnabledStatus? ETransactionAttachPDF? ETransactionPaymentEnabled? IPNSupportEnabled?
-        AllowServiceDate? AllowShipping? DefaultShippingAccount? DefaultTerms DefaultCustomerMessage
+        AllowDeposit?
+        AllowDiscount?
+        AllowEstimates?
+        AllowServiceDate?
+        AllowShipping?
+        AutoApplyCredit?
+        CustomField?
+        CustomTxnNumbers?
+        DefaultCustomerMessage
+        DefaultDiscountAccount?
+        DefaultShippingAccount?
+        DefaultTerms
+        EmailCopyToCompany?
+        EstimateMessage
+        ETransactionAttachPDF?
+        ETransactionEnabledStatus
```

Comment:
Before we had this as a Boolean but the docs show it as an `ETYransactionEnabledStatusEnum`